.travis.yml

```diff
@@ -1,3 +1,7 @@
+sudo: required
```

Comment:
These specs fail on PhantomJS 1.9, so I opted to update the Travis config to run on a VM with PhantomJS 2.0 (rather than install it from scratch on each test run).